cocoa/dlg.m

```diff
@@ -47,6 +47,9 @@ - (DlgResult)run {
 		[alert setIcon:[NSImage imageNamed:NSImageNameCaution]];
 		[alert addButtonWithTitle:@"OK"];
 		break;
+	case MSG_OK:
```

Comment:
It looks like the windows/linux dialogs include an "information" icon? Probably for consistency there should be one here:

    [alert setIcon[NSImage imageNamed:NSImageNameInfo]];

A list of image names is below, I didn't see any other candidates:
https://developer.apple.com/documentation/appkit/nsimagename?language=objc

Unfortunately I'm not in a position to test OSX changes either at present, but the changes look good so I won't let that hold things up :)

Reply:
Ah, I didn't notice this message first. I'll look into that link and see if I can include the "information" icon on OSX.